mubench.reviewsite/templates/review.phtml

```diff
@@ -76,6 +94,50 @@ use MuBench\ReviewSite\Model\Experiment;
+                <td>
+                            <? foreach($misuse->getTags() as $tag) : ?>
+                                <div class="misuse-tag">
+                                    <span><?= htmlspecialchars($tag['name']) ?></span>
+                                    <? if($user) : ?>
+                                    <div style="display:inline-block;padding-right:2px;margin:0;color:black">
+                                        <form action="<?= "{$api_url_prefix}upload/delete/tag"?>" method="post" target="">
+                                            <input type="hidden" name="exp" value="<?= htmlspecialchars($experiment->getId()); ?>">
+                                            <input type="hidden" name="detector" value="<?= htmlspecialchars($detector->id); ?>">
+                                            <input type="hidden" name="project" value="<?= htmlspecialchars($misuse->getProject()); ?>">
+                                            <input type="hidden" name="version" value="<?= htmlspecialchars($misuse->getVersion()); ?>">
+                                            <input type="hidden" name="misuse" value="<?= htmlspecialchars($misuse->id); ?>">
+                                            <input type="hidden" name="tag" value="<?= htmlspecialchars($tag['id']); ?>">
+                                            <input type="hidden" name="path" value="<?= htmlspecialchars($path); ?>">
+                                            <button style="border:none;background-color:gray;outline:none;margin:0;padding:0" type="submit"><i class="fa fa-trash"></i></button>
+                                        </form>
+                                    </div>
+                                    <? endif; ?>
+                                </div>
+                            <? endforeach; ?>
+                        <? if($user): ?>
+                        <div style="display:inline-block;height:100%;padding:4px;">
+                            <form id="tag-form" action="<?= $api_url_prefix ?>upload/tag" method="post" target="">
+                                <input type="hidden" name="exp" value="<?= htmlspecialchars($experiment->getId()); ?>">
+                                <input type="hidden" name="detector" value="<?= htmlspecialchars($detector->id); ?>">
+                                <input type="hidden" name="project" value="<?= htmlspecialchars($misuse->getProject()); ?>">
+                                <input type="hidden" name="version" value="<?= htmlspecialchars($misuse->getVersion()); ?>">
+                                <input type="hidden" name="misuse" value="<?= htmlspecialchars($misuse->id); ?>">
+                                <input type="hidden" name="path" value="<?= htmlspecialchars($path); ?>">
+                                <input id="tag-input" class="tag-input" onInput='onInput()' list="tags" name="tag" placeholder="add new tag" autocomplete="off">
```

Comment:
Please set the max length of the input to 100, to ensure it fits into the database.